mysql-test/mysql-test-run.pl

```diff
@@ -4794,74 +4794,6 @@ ($)
 	  if ($analyze){
 	    run_on_all($tinfo, "analyze-$analyze");
 	  }
-<<<<<<< HEAD
-||||||| 9dd29a1b274
-	}
-	mtr_report_test_passed($tinfo);
-      }
-      elsif ( $res == 62 )
-      {
-	# Testcase itself tell us to skip this one
-	$tinfo->{skip_detected_by_test}= 1;
-	# Try to get reason from test log file
-	find_testcase_skipped_reason($tinfo);
-	mtr_report_test_skipped($tinfo);
-	# Restart if skipped due to missing perl, it may have had side effects
-	if ( restart_forced_by_test('force_restart_if_skipped') ||
-             $tinfo->{'comment'} =~ /^perl not found/ )
-	{
-	  stop_all_servers($opt_shutdown_timeout);
-	}
-      }
-      elsif ( $res == 65 )
-      {
-	# Testprogram killed by signal
-	$tinfo->{comment}=
-	  "testprogram crashed(returned code $res)";
-	report_failure_and_restart($tinfo);
-      }
-      elsif ( $res == 1 )
-      {
-	# Check if the test tool requests that
-	# an analyze script should be run
-	my $analyze= find_analyze_request();
-	if ($analyze){
-	  run_on_all($tinfo, "analyze-$analyze");
-	}
-=======
-	}
```

Comment:
Do you know why we removed this part in the past? What if we bring it back to life?

Reply:
We have not removed this code block completely, but instead have moved this inside the loop  https://github.com/percona/percona-xtradb-cluster/blob/ccd2185ba1247b3460f6074a40ec7582ccbf4f61/mysql-test/mysql-test-run.pl#L4596. This was done as part of https://github.com/percona/percona-xtradb-cluster/commit/b1b9d5940442f75804d1b